migration-tool/src/main/resources/META-INF/rewrite/upgrade-sdk-dependencies.yml

```diff
@@ -185,6 +185,12 @@ recipeList:
       newGroupId: software.amazon.awssdk
       newArtifactId: servicecatalog
       newVersion: 2.23.16-SNAPSHOT
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: com.amazonaws
+      oldArtifactId: aws-java-sdk-timestreaminfluxdb
+      newGroupId: software.amazon.awssdk
+      newArtifactId: timestreaminfluxdb
+      newVersion: 2.23.16-SNAPSHOT
```

Comment:
These `newVersion` number are not updated? I suppose this is not an issue? I am not too familiar either with open-rewrite yet either